### ISSUE TITLE
Change Linux Elastic Premium to do run from package = 1 by default

### ISSUE THREAD

**Issue by ankitkumarr** [Azure/azure-functions-core-tools#2502]
Right now Linux Elastic Premium does `WEBSITE_RUN_FROM_PACKAGE` = url deployment by default for Linux Premium. We can and should change this to do zipdeploy using `WEBSITE_RUN_FROM_PACKAGE` = 1.

We should confirm with Jenny and the platform team to double check.

**Comment by fabiocav** [comment]
Work is complete, but pushing this to after the code/deployment freeze.

**Comment by fabiocav** [comment]
Assigning this to 103 for completion.

**Comment by fabiocav** [comment]
Waiting for confirmation from @balag0 and/or @amamounelsayed

**Comment by balag0** [comment]
Yes it is supported and there is a PR open as well - https://github.com/Azure/azure-functions-core-tools/pull/2529/files

**Comment by zbuchli** [comment]
Any movement on this PR?
